ontology/yaml/resources/HVAC/entity_types/PMP.yaml

```diff
@@ -140,6 +140,14 @@ PMP_SS_RMM:
   - low_pressure_status
   - high_pressure_status
 
+PMP_CPC_RMM:
+  description: "Pump with circulation pump and run mode monitoring."
+    is_canonical: true
+    implements:
+    - PMP
+    - CPC
+    - RMM
```

Comment:
`RMM` is typically reserved for components that perform heating/cooling. It is odd to see this used on a pump.

Reply:
I have updated to PMP_SS_RMM_CPC for now. If its prefered to not use RMM, PMP_VSC_CPC could possible work if `run_command` and `speed_percentage_command` are marked missing, but unsure if using VSC and marking `speed_percentage_command` as missing would be valid.